src/custom/api/gnosisProtocol/api.ts

```diff
@@ -255,48 +265,43 @@ export async function getOrder(chainId: ChainId, orderId: string): Promise<Order
   }
 }
 
-export async function getAppDataDoc(chainId: ChainId, address: string): Promise<AppMetadata | null> {
-  console.log(`[api:${API_NAME}] Get AppData doc for`, chainId, address)
+type GetTradesParams = {
+  chainId: ChainId
+  owner: string
+} & PaginationParams
+
+export async function getTrades(params: GetTradesParams): Promise<TradeMetaData[]> {
+  const { chainId, owner, limit, offset } = params
+  const qsParams = stringify({ owner, limit, offset })
```

Comment:
You are a bit ahead as the current `/trades` endpoint do not have pagination.
Also, pagination like this will be present on the new trades endpoint (`/api/v1/account/{owner}/trades`), similar to the new orders endpoint: https://protocol-rinkeby.dev.gnosisdev.com/api/#/default/get_api_v1_account__owner__orders

Although, that's not available yet.

All this is to say this query might not work as you expect and return all trades instead.

But that is ok, I believe the default behaviour is to ignore unknown parameters.

No need to change anything, just FYI

Reply:
Yep, kind of nice that we are already prepared, I would also expect them to ignore them if it's not implemented, and we will see it working as soon as they implement it, so not bad to leave it imo